contracts/truefi2/TrueCreditAgency.sol

```diff
@@ -271,6 +286,7 @@ contract TrueCreditAgency is UpgradeableClaimable {
     function _payInterestWithoutTransfer(ITrueFiPool2 pool) internal returns (uint256) {
         uint256 accruedInterest = interest(pool, msg.sender);
         totalPaidInterest[pool][msg.sender] = totalPaidInterest[pool][msg.sender].add(accruedInterest);
+        nextInterestRepayTime[pool][msg.sender] = block.timestamp.add(repaymentPeriod);
```

Comment:
The version of `_payInterestWithoutTransfer()` from the base branch (#778) allows partial interest repayment of a custom amount, which would be incompatible with this version since a borrower could reset their period with less than a full interest payment.

Why don't we punt this decision (where in the code we reset the interest repay time) to that PR? This PR would just remain focused on setting the interest repayment period.

You can delete this line for now and we can figure out where it best belongs in PR #778.